test/test_topography.py

```diff
@@ -928,3 +928,15 @@ def test_power_spectrum_from_profile():
     q1, C1 = t.power_spectrum_from_profile(window='hann')
 
     # TODO add check for values
+
+
+def test_equal_operator():
```

Comment:
Do you have a parallel version of the test at hand ?

Reply:
I've turned this test into a parallel one